### ISSUE TITLE
Apertium RDF - adding vartrans:relates property

### ISSUE THREAD

**Issue by jogracia** [acoli-repo/acoli-dicts#5]
In the translations, I would recommend to overload the "vartrans:source" and "vartrans:target" properties with "vartrans:relates", to allow for simpler queries (as we did in Apertium RDF v1.0).  E.g:

From : 
```
:thousand_mil-num-en-sense-mil_thousand-num-es-sense-trans a vartrans:Translation;
    vartrans:source :thousand_mil-num-en-sense;
    vartrans:target :mil_thousand-num-es-sense.
```

To 
```
:thousand_mil-num-en-sense-mil_thousand-num-es-sense-trans a vartrans:Translation;
    vartrans:source :thousand_mil-num-en-sense;
    vartrans:target :mil_thousand-num-es-sense;
    vartrans:relates :thousand_mil-num-en-sense;
    vartrans:relates :mil_thousand-num-es-sense.
```

**Comment by chiarcos** [comment]
Makes sense, but this is RDFS-inferrable (and can thus be automatically  
recovered by the user with the correct end point settings).

If we really want to put this into the data, I would suggest to provide  
the data in two versions: a "slim" version containing the information in a  
minimal representation (better for exchange) and a "full" version that  
contains inferrable information (better for querying).

There may be other pieces of duplicate information that could be dropped  
 from the slim version to save space and transfer efforts.

Best,
Christian

Am .07.2020, 13:00 Uhr, schrieb Jorge Gracia <notifications@github.com>:

**Comment by jogracia** [comment]
Having such "slim" / "full" versions of the data makes sense to me.
